audit/projects/k8s-staging-artifact-promoter/iam.json

```diff
@@ -22,6 +22,32 @@
     },
     {
       "members": [
+        "serviceAccount:service-675573440409@compute-system.iam.gserviceaccount.com"
```

Comment:
Almost certainly what happened was that you or I or someone with "enough" privs clicked on the "compute" tab while in this project (that's my default tab) and it "helpfully" enables the service.  Sigh.  We should just go and disable it in the script. so it will get cleaned up on every run.

Reply:
opened https://github.com/kubernetes/k8s.io/issues/1675

Reply:
opened https://github.com/kubernetes/k8s.io/issues/1675